boards/amd/versal2_rpu/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+#
+
+if (EXISTS "${BOARD_DIR}/${BOARD}-qemu.dts")
```

Comment:
no space between if and bracket in cmake

Reply:
will fix